docker/dev/init.dev.sh

```diff
@@ -3,6 +3,9 @@ while ! mysqladmin ping -h"$DB_HOST" --silent; do
     sleep 2
 done
 
+echo "Install dependencies"
+bundle install
```

Comment:
This is much quicker:

```suggestion
bundle check || bundle install
```

Reply:
It also has the effect of not inadvertently updating the `Gemfile.lock` and putting the Docker image out of sync.